apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/subscriber-view/edit.js

```diff
@@ -33,11 +33,15 @@ function Edit( { hasInnerBlocks, parentClientId, isSelected } ) {
 						templateLock={ false }
 						templateInsertUpdatesSelection={ false }
 						template={ [
+							[
+								'core/heading',
+								{ content: __( 'Subscriber Content', 'full-site-editing' ), level: 3 },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Subscriber-only content' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/default-visitor-content-premium-content-block)